testdata/p4_16_samples_outputs/issue1717.p4-stderr

```diff
@@ -19,7 +19,9 @@ issue1717.p4(52): [--Wwarn=uninitialized_use] warning: h1.isValid may be uniniti
 issue1717.p4(52): [--Wwarn=uninitialized_use] warning: h2.f may be uninitialized
         size = v(h1, h2);
                      ^^
-[--Wwarn=invalid_header] warning: accessing a field of an invalid header h2_0
-[--Wwarn=invalid_header] warning: accessing a field of an invalid header h1_0
-[--Wwarn=invalid_header] warning: accessing a field of an invalid header h2_0
-[--Wwarn=invalid_header] warning: accessing a field of an invalid header h1_0
+issue1717.p4(41): [--Wwarn=invalid_header] warning: accessing a field of an invalid header h2_0
+    bool b1 = h2.minSizeInBits == 32;
+              ^^
+issue1717.p4(44): [--Wwarn=invalid_header] warning: accessing a field of an invalid header h1_0
```

Comment:
Looks like it is printing the mangled name (h1_0) in the error message instead of the original name -- the original name is getting lost somewhere along the line.